libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -881,6 +881,76 @@
         "comment": "",
         "limit": 0
     },
+    "properties.canvas.rowHeight": {
+        "value": "Row height",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.canvas.rowHeight.small": {
+        "value": "small",
```

Comment:
It would be nice to provide comments for translators for these new texts as lots of these could be hard to translate in correct context in some languages.